manage.py

```diff
@@ -148,6 +148,77 @@ def cf_startup():
         subprocess.Popen(["python", "cli.py", "refresh_materialized"])
 
 
+def check_long_queries(minutes: int):
+    """
+    Check for queries running longer than interval, default is 5
+    """
+    SLACK_BOTS = "#bots"
+
+    # sets minimum minutes interval at 2
+    if minutes < 2:
+        raise ValueError("Interval must be greater than 2 minutes")
+
+    SQL = """
+        SELECT *
+        FROM pg_stat_activity
+        WHERE datname <>'rdsadmin'
+        and usename ='fec_api'
+        and lower(query) like 'select %'
+        and lower(query) not like '%refresh%'
+        and lower(query) not like '%rollback%'
+        and (now() - pg_stat_activity.query_start) >= interval :minutes
+        order by pg_stat_activity.query_start desc;
+        """
+    try:
+        results = db.engine.execute(sa.text(SQL), minutes=f"{minutes} minutes")
+        rows = (results.fetchall())
+        for row in rows:
+            logger.info(row)
+        total_rows = results.rowcount
+        slack_message = "Currently {} queries running longer than {} minutes".format(total_rows, minutes)
+        logger.info(slack_message)
+        post_to_slack(slack_message, SLACK_BOTS)
+    except Exception as error:
+        logger.exception(error)
+        slack_message = "*ERROR* long running query check failed."
+        slack_message = slack_message + "\n Error message: " + str(error)
+        post_to_slack(slack_message, SLACK_BOTS)
+
+
+def clear_long_queries(minutes: int):
+    """
+    Terminate queries running longer than interval minutes, default is 5
+    """
+    SLACK_BOTS = "#bots"
+
+    # sets minimum minutes interval at 2
+    if minutes < 2:
+        raise ValueError("Interval must be greater than 2 minutes")
+
+    SQL = """
+        SELECT pg_terminate_backend(pid)
+        FROM pg_stat_activity
+        WHERE datname <>'rdsadmin'
+        and usename ='fec_api'
+        and lower(query) like 'select %'
+        and lower(query) not like '%refresh%'
+        and lower(query) not like '%rollback%'
+        and (now() - pg_stat_activity.query_start) >= interval :minutes
+        order by pg_stat_activity.query_start desc;
+        """
+    try:
+        results = db.engine.execute(sa.text(SQL), minutes=f"{minutes} minutes")
+        total_rows = results.rowcount
+        slack_message = "Terminated {} queries running longer than {} minutes".format(total_rows, minutes)
```

Comment:
@cnlucas Just like we discussed can you include the space name in the logs here?